main.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
-const logger = require('ft-next-logger');
+const logger = require('ft-next-logger').logger;
+const fetchres = require('fetchres');
+const raven = require('raven');
+let ravenMiddleware;
+
+function sendErrorDev (err, req, res) {
+	if (err.name === fetchres.ReadTimeoutError.name) {
+		logger.error('Dependency timeout -', err);
```

Comment:
ah yes, I wasn't sure of best error formatting. ft-next-logger should document this